scripts/test.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
```

Comment:
What is this file for?

Reply:
Oops, didn't mean to commit this one haha